packages/fullstack/src/launch.rs

```diff
@@ -119,21 +119,21 @@ impl<Props: Clone + serde::Serialize + serde::de::DeserializeOwned + Send + Sync
     }
 
     #[cfg(feature = "web")]
-    fn launch_web(self) {
+    pub fn launch_web(self) {
```

Comment:
If we make these public, we need to add documentation to the function. Other than that, this looks like a good change!